src/main/io/rcdevice_cam.c

```diff
@@ -0,0 +1,278 @@
+        if (reqResult) {
+            rcdeviceInMenu = true;
+            beeper(BEEPER_CAM_CONNECTION_OPEN);
+        }
+        break;
+    case RCDEVICE_CAM_KEY_CONNECTION_CLOSE:
+        reqResult = runcamDeviceClose5KeyOSDCableConnection(camDevice);
+        if (reqResult) {
+            rcdeviceInMenu = false;
+            beeper(BEEPER_CAM_CONNECTION_CLOSE);
+        }
+        break;
+    case RCDEVICE_CAM_KEY_ENTER:
+    case RCDEVICE_CAM_KEY_LEFT:
+    case RCDEVICE_CAM_KEY_UP:
+    case RCDEVICE_CAM_KEY_RIGHT:
+    case RCDEVICE_CAM_KEY_DOWN:
+        reqResult = rcdeviceCamSimulate5KeyCablePress(key);
+        break;
+    case RCDEVICE_CAM_KEY_RELEASE:
+        reqResult = runcamDeviceSimulate5KeyOSDCableButtonRelease(camDevice);
+        break;
+    default:
+        reqResult = false;
+        break;
+    }
+
+    return reqResult;
+}
+
+static void rcdevice5KeySimulationProcess(timeUs_t currentTimeUs)
+{
+    UNUSED(currentTimeUs);
+
+#ifdef CMS
+    if (cmsInMenu) {
+        return;
+    }
+#endif
+
+    if (camDevice->serialPort == NULL) {
+        return;
+    }
+
+    rcdeviceCamSimulationKeyEvent_e key = RCDEVICE_CAM_KEY_NONE;
+
+    if (needRelease) {
+        if (IS_MID(YAW) && IS_MID(PITCH) && IS_MID(ROLL)) {
+            key = RCDEVICE_CAM_KEY_RELEASE;
+            if (!rcdeviceSend5KeyOSDCableSimualtionEvent(key)) {
+                rcdeviceInMenu = false;
+            } else {
+                needRelease = false;
+            }
+            return;
+        } else {
+            return;
+        }
+    } else {
+        if (IS_MID(THROTTLE) && IS_MID(ROLL) && IS_MID(PITCH) && IS_LO(YAW)) { // Disconnect HI YAW
+            if (rcdeviceInMenu) {
+                key = RCDEVICE_CAM_KEY_CONNECTION_CLOSE;
+            }
+        } else {
+            if (rcdeviceInMenu) {
+                if (IS_LO(ROLL)) { // Left LO ROLL
+                    key = RCDEVICE_CAM_KEY_LEFT;
+                } else if (IS_HI(PITCH)) { // Up HI PITCH
+                    key = RCDEVICE_CAM_KEY_UP;
+                } else if (IS_HI(ROLL)) { // Right HI ROLL
+                    key = RCDEVICE_CAM_KEY_RIGHT;
+                } else if (IS_LO(PITCH)) { // Down LO PITCH
+                    key = RCDEVICE_CAM_KEY_DOWN;
+                } else if (IS_MID(THROTTLE) && IS_MID(ROLL) && IS_MID(PITCH) && IS_HI(YAW)) { // Enter HI YAW
+                    key = RCDEVICE_CAM_KEY_ENTER;
+                }
+            } else {
+                if (IS_MID(THROTTLE) && IS_MID(ROLL) && IS_MID(PITCH) && IS_HI(YAW) && !ARMING_FLAG(ARMED)) { // Enter HI YAW
+                    key = RCDEVICE_CAM_KEY_CONNECTION_OPEN;
+                }
+            }
+        }
+    }
+
+    if (key != RCDEVICE_CAM_KEY_NONE) {
+        if (!rcdeviceSend5KeyOSDCableSimualtionEvent(key)) {
+            rcdeviceInMenu = false;
+        } else {
+            needRelease = true;
+        }
+    }
+}
+
+void rcdeviceUpdate(timeUs_t currentTimeUs)
+{
+    UNUSED(currentTimeUs);
```

Comment:
Remove `UNUSED` since `currentTimeUs` is used.

Reply:
yes..